packages/@react-aria/aria-modal-polyfill/src/index.ts

```diff
@@ -32,7 +32,7 @@ export function watchModals(selector:string = 'body'): void {
   let observer = new MutationObserver((mutationRecord) => {
     for (let mutation of mutationRecord) {
       if (mutation.type === 'childList' && mutation.addedNodes.length > 0) {
-        let addNode = Array.from(mutation.addedNodes).find((node: Element) => node.querySelector('[aria-modal="true"], [data-ismodal]')) as HTMLElement;
+        let addNode: Element = (Array.from(mutation.addedNodes).find((node: any) => node.querySelector?.('[aria-modal="true"], [data-ismodal]')) as HTMLElement);
```

Comment:
o, i think we did some of these wrong possibly 
https://github.com/adobe/react-spectrum/blob/main/packages/@react-aria/focus/src/FocusScope.tsx#L401
https://github.com/adobe/react-spectrum/blob/main/packages/%40react-aria/utils/src/filterDOMProps.ts#L54

still getting used to the `?.` shorthand

Reply:
> o, i think we did some of these wrong possibly

Why?

Reply:
`node.querySelector?.(`
vs
`node?.querySelector(`
I think we just goofed on it, just happen to remember writing one

anyhow, i don't think it's doing harm right now, we could fix that in another PR

Reply:
I think the functions you mentioned are fine. In this case, the MutationObserver returns a [Node](https://developer.mozilla.org/de/docs/Web/API/Node), not an [Element](https://developer.mozilla.org/de/docs/Web/API/Element). Nodes have no `querySelector` method.

So `node.querySelector?.(...)` is equivalent to `node.querySelector ? node.querySelector(...) : null` or `node instanceof Element && node.querySelector(...)`